go/worker/sentry/grpc/init.go

```diff
@@ -29,9 +32,8 @@ const (
 
 	// CfgUpstreamAddress is the grpc address of the upstream node.
 	CfgUpstreamAddress = "worker.sentry.grpc.upstream.address"
-
-	// CfgUpstreamCert is the path to the certificate files of the upstream node.
-	CfgUpstreamCert = "worker.sentry.grpc.upstream.cert"
+	// CfgUpstreamID is the node ID of the upstream node.
+	CfgUpstreamID = "worker.sentry.grpc.upstream.id"
```

Comment:
Label the PR with `breaking-cfg` and include the change in the changelog fragment.

Reply:
Will do.